utils/version_info.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2025 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "version_info.h"
+
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+#include "absl/status/statusor.h"
+#include "absl/strings/str_format.h"
+
+namespace
+{
+// These are relative to install/
+static constexpr std::string_view
+kDiveAndroidLibrariesBuildTypePath = "TARGETTING_ANDROID_BUILD_TYPE.txt";
+static constexpr std::string_view kProfilingPluginShaPath = "dive_profiling_plugin/SHA";
```

Comment:
nit: `static` (in this context) can be removed since the anonymous namespace already instructs the compiler to use internal linkage.
```suggestion
constexpr std::string_view
kDiveAndroidLibrariesBuildTypePath = "TARGETTING_ANDROID_BUILD_TYPE.txt";
constexpr std::string_view kProfilingPluginShaPath = "dive_profiling_plugin/SHA";
```